components/Main/Main.js

```diff
@@ -0,0 +1,17 @@
+// this component will contain all the content components besides the Header and footer. For now it will only contain a short welcome text and a link for a better navigation.
+
+import Link from "next/link";
+
+export default function Header() {
+  return (
+    <>
+      <h1>Willkommen zu Movie Flip 🎬</h1>
+      <h2>Deine Filmempfehlung in vier einfachen Schritten</h2>
+      <Link href="/filmEmpfehlung">
```

Comment:
If you prefer, you can write the information which is relevant for the user in German, but everything on the developer site, should be in english, to keep it coherent:)

```suggestion
      <Link href="/movieRecommendations">
```

Reply:
done.